anta/cli/exec/utils.py

```diff
@@ -90,18 +86,14 @@ async def collect(dev: AntaDevice, command: str, outformat: Literal["json", "tex
     res = await asyncio.gather(*coros, return_exceptions=True)
     for r in res:
         if isinstance(r, Exception):
-            logger.error(f"Error when collecting commands: {str(r)}")
+            logger.error(f"Error when collecting commands: {r!s}")
```

Comment:
why not - though we may not all be format shortcut savvy

Reply:
https://docs.astral.sh/ruff/rules/explicit-f-string-type-conversion/